src/gui/aboutdialog.cpp

```diff
@@ -20,10 +20,11 @@
 namespace OCC {
 
 AboutDialog::AboutDialog(QWidget *parent)
-    : QDialog(parent)
+    : QDialog(parent, Qt::Sheet)
     , ui(new Ui::AboutDialog)
 {
-    setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+    setWindowModality(Qt::ApplicationModal);
+    setWindowFlag(Qt::WindowContextHelpButtonHint, false);
```

Comment:
Both of these have been duplicated a lot now. Should we introduce a helper? Maybe `Qt::Sheet` can be set that way as well.

Reply:
I was tempted to use raiseDialog for that 🙈

Reply:
Yeah, please let's add more functionality to it that isn't covered by the name and not really documented...

I think `raiseDialog` is also used in places where those flags are not needed, though.